src/components/checkbox/checkbox.scss

```diff
@@ -43,6 +43,7 @@
 
     label {
         line-height: normal;
+        margin-left: -0.25rem;
```

Comment:
There is already this `padding-left`:

![image](https://user-images.githubusercontent.com/154725/118465594-52afc280-b702-11eb-9f8a-607fc759684f.png)

I'm thinking maybe it's better to zero that out instead? 🤔

Reply:
Good catch. You're right!

Reply:
will you do or should I?

Reply:
Did you check that your fixup works? Because I don't think it will. `.mdc-form-field label` has lower specificity than `.mdc-form-field > label`.

Reply:
yes I did :) it works

Reply:
![image](https://user-images.githubusercontent.com/154725/118477967-9d840700-b70f-11eb-8ebb-0bdc784510a1.png)

Indeed it does… but how the actual f… 🤨

I wonder if this is how physicists feel when they discover "new physics"… 😂

Reply:
CSS weights man… I never understand them 😄